src/test/java/org/sagebionetworks/bridge/models/schedules2/adherence/weekly/WeeklyAdherenceReportTest.java

```diff
@@ -31,48 +31,65 @@ public void canSerialize() {
         report.setTestAccount(true);
         report.setWeeklyAdherencePercent(79);
+        report.setRows(ImmutableList.of(row));
         report.setByDayEntries(ImmutableMap.of(
                 new Integer(6), ImmutableList.of(new EventStreamDay())));
         report.setNextActivity(nextActivity);
-        
+
         // It's there, it works, it's persisted, but it's not part of JSON output
         assertTrue(report.isTestAccount());
         
         // These are not in the JSON
         assertEquals(report.getAppId(), TEST_APP_ID);
         assertEquals(report.getStudyId(), TEST_STUDY_ID);
         assertEquals(report.getUserId(), TEST_USER_ID);
-        assertEquals(report.getLabels(), ImmutableSet.of("label1", "label2"));
+        assertEquals(report.getSearchableLabels(), ImmutableSet.of("label1", "label2"));
         
         JsonNode node = BridgeObjectMapper.get().valueToTree(report);
-        assertEquals(node.size(), 8);
+        
+        assertEquals(node.size(), 9);
         assertNull(node.get("appId"));
         assertNull(node.get("studyId"));
         assertNull(node.get("userId"));
         assertEquals(node.get("clientTimeZone").textValue(), TEST_CLIENT_TIME_ZONE);
         assertEquals(node.get("createdOn").textValue(), MODIFIED_ON.withZone(DateTimeZone.forID(TEST_CLIENT_TIME_ZONE)).toString());
-        assertEquals(node.get("rowLabels").get(0).textValue(), "label1");
-        assertEquals(node.get("rowLabels").get(1).textValue(), "label2");
         assertEquals(node.get("weeklyAdherencePercent").intValue(), 79);
         assertEquals(node.get("participant").get("identifier").textValue(), TEST_USER_ID);
-        assertNull(node.get("accountTest"));
+        assertTrue(node.get("testAccount").booleanValue());
         assertEquals(node.get("nextActivity").get("type").textValue(), "NextActivity");
         assertEquals(node.get("byDayEntries").get("6").get(0).get("type").textValue(), "EventStreamDay");
         assertEquals(node.get("type").textValue(), "WeeklyAdherenceReport");
-        
-        report.setParticipant(new AccountRef(account, "study1"));
-        report.setByDayEntries(ImmutableMap.of(new Integer(6), ImmutableList.of()));
-        report.setNextActivity(nextActivity);
+
+        assertEquals(node.get("rows").size(), 1);
+        JsonNode rowNode = node.get("rows").get(0);
+        assertEquals(rowNode.get("label").textValue(), "rowLabel");
+        assertEquals(rowNode.get("searchableLabel").textValue(), ":rowLabel:");
+        assertEquals(rowNode.get("sessionGuid").textValue(), "sessionGuid");
+        assertEquals(rowNode.get("sessionSymbol").textValue(), "sessionSymbol");
+        assertEquals(rowNode.get("sessionName").textValue(), "sessionName");
+        assertEquals(rowNode.get("studyBurstId").textValue(), "studyBurstId");
+        assertEquals(rowNode.get("studyBurstNum").intValue(), 2);
+        assertEquals(rowNode.get("week").intValue(), 4);
```

Comment:
The only field that's not included here is startEventId. Might be worth including it, though it is already tested in WeeklyAdherenceReportRowTest.